### ISSUE TITLE
A11y_SQLAzureDataStudio_SchemaCompare_MenuBar_Keyboard: Tool tip is not visible when content is navigated using keyboard.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9402]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #SQL Azure Data Studio; #DesktopApp; #Win32; #MAS1.1.1; #Benchmark; #A11y_SQLAzureDataStudio; #A11ySev2; #Keyboard; #MAS2.1.1;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS reference: MAS2.1.1,MAS1.1.1

**Repro Steps:**
1. Launch Azure Data Studio application.
2. Connect to server with windows credentials.
3. Navigate to Extensions from left menu.
4. Search & install Schema Compare extension.
5. Navigate to Connections from left menu.
6. Expand the server connected in step 2.
7. Right click an on prem database and select Schema compare from Context menu.
8. Navigate till Select Source or Select Target & hit enter.
9. Provide all the details for Source & Target.
10. Navigate till Schema Compare Menu bar below Schema Compare tab.
11. Check if tool tip is visible for all the controls in the menu bar.

**Actual:**
Tool tip is not visible for the controls when we navigate to the controls using keyboard

**Expected:**
Tool tip should be visible when the content is navigated using keyboard also.

**User Impact:**
Low vision users will not receive information that is present in the tool tip.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:**
**MAS1.1.1:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}
**MAS2.1.1:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment link reference:**
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-atkulk_microsoft_com/EXSB9PxSFYVIqv_uiaoUuxoB2MLKxxRvZeqC43nSoYja1g?e=32webg

**Comment by alanrenmsft** [comment]
umbrella bug: https://github.com/Microsoft/vscode/issues/52016

**Comment by hannahqin** [comment]
This is also a duplicate of an upstream bug that was previously approved for exclusion: http://sqlbuvsts01:8080/Main/SQL%20Server/_workitems#_a=edit&id=11697790. It is already documented in our Conformance Document.

**Comment by Vishnuvarthancr7** [comment]
GitHubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
